__tests__/__integration__/imperative/__tests__/__integration__/cli/config/import/cli.imperative-test-cli.config.import.integration.test.ts

```diff
@@ -0,0 +1,324 @@
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+        });
+
+        describe("from the disk", () => {
+            it("should fail to import a config from a bad path", () => {
+                const response = runCliScript(path.join(__dirname, "/__scripts__/import_config.sh"), TEST_ENVIRONMENT.workingDir, [
+                    path.join(__dirname, "__resources__", "__fake__", "test.config.good.with.schema.json"),
+                    "--user-config false --global-config false"
+                ]);
+
+                expect(response.stdout.toString()).toEqual("");
+                expect(response.stderr.toString()).toContain("no such file or directory");
+                expect(response.status).toEqual(1);
+
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json"))).toEqual(false);
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+
+            it("should fail to import a schema from a bad path", () => {
+                const response = runCliScript(path.join(__dirname, "/__scripts__/import_config.sh"), TEST_ENVIRONMENT.workingDir, [
+                    path.join(__dirname, "__resources__", "test.config.bad.with.schema.json"),
+                    "--user-config false --global-config false"
+                ]);
+
+                expect(response.stderr.toString()).toContain("Failed to download schema");
+                expect(response.status).toEqual(0);
+
+                expectFilesAreEqual(path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json"),
+                    path.join(__dirname, "__resources__", "test.config.bad.with.schema.json"));
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+
+            it("should fail to import a config that is invalid JSON from a path", () => {
+                const response = runCliScript(path.join(__dirname, "/__scripts__/import_config.sh"), TEST_ENVIRONMENT.workingDir, [
+                    path.join(__dirname, "__resources__", "test.config.bad.json"),
+                    "--user-config false --global-config false"
+                ]);
+
+                expect(response.stdout.toString()).toEqual("");
+                expect(response.stderr.toString()).toContain("Unexpected end of JSON input");
+                expect(response.status).toEqual(1);
+
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json"))).toEqual(false);
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+
+            it("should fail to import a schema if it is defined with an absolute path", () => {
```

Comment:
Just wondering why we refuse to import a schema that has an absolute path?

Reply:
Discussed this in person and made some updates to command description, output, and test names to clarify why a schema is or is not imported.